Content.Server/Medical/Components/CryoPodComponent.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Content.Server.Atmos;
+using Content.Shared.Atmos;
+using Robust.Shared.Containers;
+
+namespace Content.Server.Medical.Components;
+
+[RegisterComponent]
+public sealed class CryoPodComponent: Component, IGasMixtureHolder
+{
+    /// <summary>
+    /// Specifies the name of the atmospherics port to draw gas from.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("port")]
+    public string PortName { get; set; } = "port";
+
+    /// <summary>
+    /// Local air buffer that will be mixed with the pipenet, if one exists, per tick.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("gasMixture")]
+    public GasMixture Air { get; set; } = new(Atmospherics.OneAtmosphere);
+
+    /// <summary>
+    /// Container for mobs inserted in the pod.
+    /// </summary>
+    public ContainerSlot BodyContainer = default!;
+
+    /// <summary>
+    /// Specifies the name of the atmospherics port to draw gas from.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("solutionContainerName")]
+    public string SolutionContainerName { get; set; } = "beakerSlot";
+
+    /// <summary>
+    /// Accumulator used to keep track of when to inject chemicals from the inserted beaker to the mob inside the pod.
+    /// </summary>
+    [DataField("accumulator")]
+    public float Accumulator = 0f;
```

Comment:
I think this instead should be turned into a target TimeSpan to avoid accumulators. See my comment in the CryoPodSystem update() method.